Realm/Realm/DatabaseTypes/RealmValue.cs

```diff
@@ -149,6 +216,33 @@ private RealmValue(IRealmObjectBase obj) : this()
         [EditorBrowsable(EditorBrowsableState.Never)]
         public static RealmValue Object(IRealmObjectBase value) => new(value);
 
+        /// <summary>
+        /// Gets a RealmValue representing a list.
+        /// </summary>
+        /// <param name="value"> The input list to copy. </param>
+        /// <returns> A new RealmValue representing the input list. </returns>
+        /// <remarks> Once created, this RealmValue will just wrap the input collection.
+        /// After the object containing this RealmValue gets managed this value will be a Realm list.</remarks>
+        public static RealmValue List(IList<RealmValue> value) => new(value);
```

Comment:
Yes

Reply:
Thinking about this... We probably shouldn't.
The thing is that then we'd have a different behaviour between managed and unmanaged `RealmValue`, because we can't cast the `IEnumerable` to `IList` and so users can't use index access for example.

Reply:
Yeah, good point.